vm_dump.c

```diff
@@ -958,6 +958,28 @@ rb_dump_machine_register(const ucontext_t *ctx)
         dump_machine_register(mctx->__gregs[REG_S2+7], "s9");
         dump_machine_register(mctx->__gregs[REG_S2+8], "s10");
         dump_machine_register(mctx->__gregs[REG_S2+9], "s11");
+#   elif defined __loongarch64
+        dump_machine_register(mctx->__gregs[LARCH_REG_SP], "sp");
+        dump_machine_register(mctx->__gregs[LARCH_REG_S0], "s0");
+        dump_machine_register(mctx->__gregs[LARCH_REG_S1], "s1");
+        dump_machine_register(mctx->__gregs[LARCH_REG_A0], "a0");
+        dump_machine_register(mctx->__gregs[LARCH_REG_A0+1], "a1");
+        dump_machine_register(mctx->__gregs[LARCH_REG_A0+2], "a2");
+        dump_machine_register(mctx->__gregs[LARCH_REG_A0+3], "a3");
+        dump_machine_register(mctx->__gregs[LARCH_REG_A0+4], "a4");
+        dump_machine_register(mctx->__gregs[LARCH_REG_A0+5], "a5");
+        dump_machine_register(mctx->__gregs[LARCH_REG_A0+6], "a6");
+        dump_machine_register(mctx->__gregs[LARCH_REG_A0+7], "a7");
+        dump_machine_register(mctx->__gregs[LARCH_REG_A0+7], "a7");
+        dump_machine_register(mctx->__gregs[LARCH_REG_S0], "s0");
+        dump_machine_register(mctx->__gregs[LARCH_REG_S0+1], "s1");
```

Comment:
Why do `s0` and `s1` dump twice?
I think `LARCH_REG_S0+1 == LARCH_REG_S1`, right?

Reply:
Thank you very much for pointing out my problem, I will change this.
LARCH_REG_S0+1 == LARCH_REG_S1 is correct.